packages/core-plugin-api/src/apis/definitions/FeatureFlagsApi.ts

```diff
@@ -23,7 +23,8 @@ import { ApiRef, createApiRef } from '../system';
  */
 export type FeatureFlag = {
   name: string;
-  pluginId: string;
+  pluginId?: string;
```

Comment:
Why now make this optional?

Reply:
Because app level feature flags do not have pluginId